src/netaddress.cpp

```diff
@@ -422,6 +435,11 @@ bool CNetAddr::IsI2P() const { return m_net == NET_I2P; }
  */
 bool CNetAddr::IsCJDNS() const { return m_net == NET_CJDNS; }
 
+/**
+ * Check whether this object represents a Yggdrasil address.
```

Comment:
shauld be `an Yggdrasil address`

Reply:
Done in aa5c9779bdd1c816899c0704ee1d965a57d6f036